src/components/common/ProductBox/ProductBox.js

```diff
@@ -49,7 +49,8 @@ const ProductBox = ({ name, price, promo, stars, image }) => (
       </div>
       <div className={styles.price}>
         <Button noHover variant='small'>
-          $ {price}
+          {oldPrice != 0 && <span className={styles.oldPrice}>$ {oldPrice} </span>}${' '}
```

Comment:
Hmm... Why don't we just use `oldPrice` as condition? `{oldPrice && <span className={styles.oldPrice}>$ {oldPrice} </span>}${' '}`. It should work the same do't you think? :)

Reply:
that was my first idea but it dosen't work the same 
![contidional statment](https://user-images.githubusercontent.com/52444354/70821903-8e8f7b80-1ddc-11ea-9417-4ffad664345e.gif)

Reply:
i think we can leave it as it is

Reply:
Okey. So let it be. Resolve conflicts and we are good to go :)